lib/iris/common/mixin.py

```diff
@@ -5,43 +5,20 @@
 # licensing details.
 
```

Comment:
@lbdreyer My intent here was simple, as in, it's a namespace for common mixin classes to live.

It might just be the case that `CFVariableMixin` will be our only mixin class, but at least we have scope for collecting such classes together in one place, rather than having them sprinkled around the codebase. HTH :wink: